.github/workflows/canary-publish.yml

```diff
@@ -0,0 +1,28 @@
+name: Deploy a Canary Build
+on:
+  workflow_dispatch:
+    branches-ignore:
+      - master
```

Comment:
I'm starting to think it'd be good to have a canary branch to auto deploy from anyway.

Reply:
We could do that as well! I don't think it hurts to have an ad-hoc deploy available for this, but if we had a branch that was always auto-deployed, I don't think that would be bad.

Sometimes we'll have more than one PR being reviewed, and I think having this workflow manually dispatchable will make it easier to test those things as we're reviewing them.

Reply:
Another option would be to have a workflow that creates a tag for the branch, and then we could test based on branch tags. (So like `npm install es-components@hot-new-feature-branch`.)